pkg/reconciler/v1alpha1/taskrun/entrypoint/entrypoint_test.go

```diff
@@ -1,4 +1,4 @@
-package entrypoint_test
+package entrypoint
```

Comment:
Any reason to use `entrypoint` here (instead of `entrypoint_test`) ? I'm guessing for `cacheSize` :upside_down_face: 
(I tend to really like when the test package is different from the current package as it forces to write test that tests the behavior more than the implementation)

Reply:
> I actually usually prefer to have tests in the same package, so that package-level vars/consts/funcs don't get exported just so they can be called from tests.

Yeah, we should definitely not export var/const/func just for the sake of tests :wink: